src/main/java/com/vinylteam/vinyl/service/impl/DefaultUserPostService.java

```diff
@@ -0,0 +1,28 @@
+package com.vinylteam.vinyl.service.impl;
+
+import com.vinylteam.vinyl.dao.UserPostDao;
+import com.vinylteam.vinyl.entity.UserPost;
+import com.vinylteam.vinyl.service.UserPostService;
+import com.vinylteam.vinyl.util.MailSender;
+
+public class DefaultUserPostService implements UserPostService {
+    private final UserPostDao userPostDao;
```

Comment:
For future. Please add empty row after class declaration and before closing it. It is kind of our code style